### DISCUSSION TITLE
How to allow other docker containers to access private network subnet via gluetun container?

### DISCUSSION THREAD

**Discussion by beeyev** [qdm12/gluetun#1350] [Q&A]
Simple situation.
I have a private network with subnet `192.168.1.0/24` inside this subnet I have some servers, for example file server with IP address `192.168.1.12` and database server with ip `192.168.1.14`

I have a working OpenVPN configuration server, and it works fine, when I initialize the connection from my local machine to this server. I can easily access any host inside the remote network  `192.168.1.0/24`.

The question:
I have multiple docker containers in the same docker network, here is a part of my `docker-compose` file
```yaml
networks:
  network1:
    driver: bridge
```
One of these containers is this `gluetun` which successfully connects to the remote network. 
How can I tell other containers, that whenever they need to access some hostname from the network subnet `192.168.1.0/24`, they should route traffic through the  `gluetun` ? Otherwise, traffic should not route through `gluetun`, it means that  `gluetun` is not a default gateway for other containers, it only allows accessing this particular subnet `192.168.1.0/24`

**Comment by mkeguy106** [comment]
If you didn't find the answer for this.. you make aliases for your apps that use the gluetun's network.  The other apps need to be in the same bridged network as gluetun.  Then within those other apps, just reference the alias you put on the gluetun container.
